RELEASE.md

```diff
@@ -0,0 +1,54 @@
+# Releases
+Releases will be created in this repository for all official builds of `aurae` and associated projects.
+
+## Versioning
+Stable versions will follow a `vX.Y.Z` format. Versions will be consistent across everything being released.
+
+- `X`: A major version - This represents a significant change in the project. It can include breaking changes and new features.
+- `Y`: Minor version - This represents new features and improvements to the project. It can also include non-breaking changes.
+- `Z`: Patch version - This represents bug fixes and other small changes to the project.
+
+Code that has not reached stability will have an additional segment on the version.
+
+- `alpha` - Alpha releases will be built from the `main` branch of each project when the release is created. They are intended for testing and development purposes.
+
+- `beta` - Beta releases will be built from the `beta` branch of each project when the release is created. They can be used as a staging ground for releases and intended for testing and feedback from the community.
+
+## Release schedule
+| Release | Schedule |
+|---|---|
+|  alpha | weekly  | 
```

Comment:
i'd consider "HEAD" to be alpha, then a beta release (draft release) to be made _as necessary_ rather than weekly, would be enough.

having a release cut every week, and determining whether it needs to bump the minor version, is a lot of overhead for now as we're iterating on the code so much.

Reply:
Would an alpha release in this project be built every time there is a merge to `main` on one of the repositories we're making release artifacts for?

Reply:
I've written up what it might look like if alpha was released nightly. Perhaps this is too much in the other direction though.